lab-nicholasc/__test__/server.test.js

```diff
@@ -68,15 +68,13 @@ describe('api/notes', () => {
         expect(response.status).toEqual(400);
       });
   });
-
-
   test('get route /api/notes should respond with a 400 status code if there is a bad route', () => {
     return superagent.get(`${headUrl}/badroute`)
       .catch(response => {
         expect(response.status).toEqual(400);
       });
   });
-  test('get route /api/notes?id should  respond with a 500 status code if there is a bad id', () => {
+  test('get route /api/notes?id should respond with a 500 status code if there is a bad id', () => {
```

Comment:
Nick - 500 status code is an internal server error, you should be expecting a 404 for a bad id or an id that doesn't exist.